src/screens/OtherProfile/index.js

```diff
@@ -311,7 +232,7 @@ const OtherProfile = () => {
 
   const __renderListHeader = () => {
     const __renderBlockIcon = () => {
-      if (blockStatus.blocker)
+      if (isBlocking)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `if (isBlocking)` replaces the previous `if (blockStatus.blocker)`. Ensure that `isBlocking` is equivalent to `blockStatus.blocker` to avoid any behavioral changes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->